src/View/Helper/ToolbarHelper.php

```diff
@@ -57,11 +60,38 @@ public function setSort($sort)
         $this->sort = $sort;
     }
 
+    /**
+     * Dump an array of nodes
+     *
+     * @param \Cake\Error\Debug\NodeInterface[] $nodes An array of dumped variables.
+     *   Variables should be keyed by the name they had in the view.
+     * @return string Formatted HTML
+     */
+    public function dumpNodes($nodes)
```

Comment:
```suggestion
    public function dumpNodes(array $nodes): string
```

The code below has an `in_array()` check but the param doc say it's an array. Should always be an array, right?